release/release_logs/2.0.0/stress_tests/stress_test_many_tasks.json

```diff
@@ -3,45 +3,45 @@
         {
             "perf_metric_name": "stage_0_time",
             "perf_metric_type": "LATENCY",
-            "perf_metric_value": 6.4732019901275635
+            "perf_metric_value": 6.755552053451538
         },
         {
             "perf_metric_name": "stage_1_avg_iteration_time",
             "perf_metric_type": "LATENCY",
-            "perf_metric_value": 20.34087462425232
+            "perf_metric_value": 24.18174421787262
         },
         {
             "perf_metric_name": "stage_2_avg_iteration_time",
             "perf_metric_type": "LATENCY",
-            "perf_metric_value": 53.219086503982545
+            "perf_metric_value": 69.99567422866821
         },
         {
             "perf_metric_name": "stage_3_creation_time",
             "perf_metric_type": "LATENCY",
-            "perf_metric_value": 0.05356264114379883
+            "perf_metric_value": 0.05683565139770508
         },
         {
             "perf_metric_name": "stage_3_time",
             "perf_metric_type": "LATENCY",
-            "perf_metric_value": 2350.621628522873
+            "perf_metric_value": 3000.6305317878723
```

Comment:
https://b534fd88.us1a.app.preset.io/superset/explore/p/54X9bavAyEQ/

bitset seems to be within
- tight bound: `git log --oneline --ancestry-path 9ff23cd~1..71bb155`
- loose bound: `git log --oneline --ancestry-path 9ff23cd~1..c8bfd1a9cd4fb7b67f55c18555257d5f53647d44`